### PR TITLE
Asynchronous address book

### PR DESCRIPTION
**Pull request by lipis**
- https://github.com/wireapp/node-addressbook/pull/3
- https://github.com/wireapp/wire-webapp/pull/648

### CODE REVIEW

File: electron/package.json [hunk 1]
```diff
@@ -26,6 +26,6 @@
     "winston": "https://github.com/wireapp/winston.git#2.2.0-e"
   },
   "optionalDependencies": {
-    "node-addressbook": "https://github.com/wireapp/node-addressbook.git#1.0.1"
+    "node-addressbook": "https://github.com/digital-flowers/node-addressbook.git#master"
```
Comment (bennyn) [inline electron/package.json]: We should merge digital-flowers' changes back into wireapp's repo.

File: electron/package.json [hunk 2]
```diff
@@ -1,11 +1,11 @@
 {
-  "name": "wire",
-  "productName": "Wire",
+  "name": "wireinternal",
```
Comment (bennyn) [inline electron/package.json]: Is this change on purpose?

**Comment by lipis** [comment]
@bennyn Check again..
